bolt-simple-taproot.md

```diff
@@ -769,21 +766,22 @@ The to remote output has the following form:
   * `OP_1 to_remote_output_key`
   * where:
     * `taproot_nums_point = 0245b18183a06ee58228f07d9716f0f121cd194e4d924b037522503a7160432f15`
-    * `to_remote_output_key = taproot_nums_point + tagged_hash("TapTweak", taproot_nums_point|| to_remote_script_root`
+    * `to_remote_output_key = taproot_nums_point + tagged_hash("TapTweak", taproot_nums_point || to_remote_script_root)`
```

Comment:
FWIW, the NUMs point here will soon be removed (will just be the multi-sig script).